sql/core/src/test/scala/org/apache/spark/sql/execution/command/PlanResolutionSuite.scala

```diff
@@ -961,43 +961,6 @@ class PlanResolutionSuite extends AnalysisTest {
     assert(parsed4.isInstanceOf[DescribeTableCommand])
   }
 
-  test("DESCRIBE AS JSON relation") {
```

Comment:
We usually don't add analysis unit test for v2 commands, as they share the same framework which have been battle tested already.